WcaOnRails/app/webpacker/components/DelegateTab/RoleForm.jsx

```diff
@@ -32,6 +32,9 @@ function RoleForm({
   const [formValues, setFormValues] = React.useState(values.status ? delegateDetails : {});
   const [error, setError] = React.useState(false);
   const { save, saving } = useSaveAction();
+  const buttonDisabled = (disabled
+  || saving
+  || JSON.stringify(formValues) === JSON.stringify(savedValues));
```

Comment:
This may not be the most perfect way to check that these values are the same since the order of keys can be different.